opsdroid/core.py

```diff
@@ -335,7 +372,17 @@ def _connector_names(self):  # noqa: D401
                     await database.connect()
 
     async def run_skill(self, skill, config, message):
-        """Execute a skill."""
+        """Execute a skill.
+
+        Attempts to run the skill parsed and provides other arguments to the skill if necessary.
+        Also handles the exception encountered if th e
+
+        Args:
+            skill: name of the skill to be run.
+            config: The configuration the skill must be loaded in.
+            message: Message to be parsed to the chat service.
```

Comment:
@jacobtomlinson I'm not so sure about the role of the message argument. Some guidance on this would be most appreciated!

Reply:
The `message` argument here will be passed to the skill that is being run. We should probably rename `message` to `event` as we have expanded opsdroid to handle more than just text-based messages.

Reply:
Ahh, I get it now, thanks! Do you want me to rename it here?

Reply:
Sure!

Reply:
I have made the changes as advised by you. Kindly take a look, thanks!